client/src/components/Conversations/Convo.tsx

```diff
@@ -185,7 +194,11 @@ export default function Conversation({ conversation, retainView, toggleNav, isLa
             )}
           />
         ) : (
-          <div className="absolute bottom-0 right-0 top-0 w-2 bg-gradient-to-l from-0% to-transparent group-hover:w-1 group-hover:from-60%" />
+          <div
+            className={`absolute bottom-0 right-0 top-0 w-20 rounded-r-lg bg-gradient-to-l ${`from-gray-50 dark:from-[#181818] ${
+              isHovering ? 'from-60%' : 'from-0%'
```

Comment:
Thanks for the PR @walbercardoso 

is the listener necessary? because this should be the equivalent of 

`group-hover:from-60% from-0%`